QuickBytes/src/main/java/com/quickbytes/repository/RequestItemRepository.java

```diff
@@ -0,0 +1,31 @@
+package com.quickbytes.repository;
+
+import java.util.List;
+
```

Comment:
If having a consistent path variable matters, review this.

Reply:
We can worry about env variables after getting functionalities done, good suggestion though.